cnctcli/actions/products/constants.py

```diff
@@ -19,6 +19,67 @@
     'M': 'Modified',
 }
 
+PARAMS_COLS_HEADERS = {
+    'A': 'Verbose ID',
+    'B': 'ID',
+    'C': 'Action',
+    'D': 'Title',
+    'E': 'Description',
+    'F': 'Phase',
+    'G': 'Scope',
+    'H': 'Type',
+    'I': 'Required',
+    'J': 'Unique',
+    'K': 'Hidden',
+    'L': 'JSON Properties',
+}
+
+MEDIA_COLS_HEADERS = {
+    'A': 'Position',
+    'B': 'ID',
+    'C': 'Action',
+    'D': 'Type',
+    'E': 'Image File',
+    'F': 'Video URL Location',
+}
+
+CAPABILITIES_COLS_HEADERS = {
+    'A': 'Capability',
+    'B': 'Action',
+    'C': 'Value',
+}
+
+STATIC_LINK_HEADERS = {
+    'A': 'Type',
+    'B': 'Tile',
+    'C': 'Action',
+    'D': 'Url',
+}
+
+TEMPLATES_HEADERS = {
+    'A': 'ID',
+    'B': 'Title',
+    'C': 'Action',
+    'D': 'Scope',
+    'E': 'Type',
+    'F': 'Content',
+}
+
+PARAM_TYPES = [
+    "email",
```

Comment:
Please don't mix quote styles unless necessary.
Better to have a uniform style across the whole project.